.jenkins/build_docker_images.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Set variable to allow parallel building in the Docker image creation
+export DOCKER_BUILDKIT=1
+
+# Get the dockerfiles for the images to be tested on Piz Daint
+dockerfiles=( $( ls docker/Dockerfile.* ) )
+
+for df in ${dockerfiles[@]}; do
+
+   # Grab the architecture tag from the dockerfile filename
+   arch=${df:18}
+   container=us.gcr.io/vcm-ml/fv3gfs-compiled:$arch
```

Comment:
This works for a first version, but I wonder if we should try to get this container name from the Makefile/repo rather than hardocding the. name at some point